modules/pages-and-forms/pages/pro-declare-interface-in-bonita.adoc

```diff
@@ -0,0 +1,27 @@
+= Declare the interface in Bonita
+:page-aliases: ROOT:pro-declare-interface-in-bonita.adoc
+:description: This page defines the steps to declare your interface and display it in Bonita.
+
+{description}
+
+== Declare the interface in Bonita application descriptor
+
+The xref:applications:application-creation.adoc[Bonita application descriptor] is a file that let you describe all your applications.
+
+To declare a Bonita UI Designer Pro interface in it, edit it in Studio using the "source" tab.
+In this tab, you will be able to declare your application built and packaged with Bonita UI Designer Pro, using auto-completion with a "advancedApplication" XML tag.
+
+image::images/advanced-application-descriptor.png[Application Descriptor Editor]
+
+In this example, the "Advanced Overview" application is declared with the tag:
```

Comment:
**nitpick:** 
For blind people, it's not clear whether to add this snippet code inside the `<applications></applications>` tag.